### PR TITLE
Feat:Input and navbar added

### PR DESCRIPTION
**Pull request by ArpitPatni**
Hey guys please review my input and navbar.

### CODE REVIEW

**Review by sumitch29 — APPROVED** [review]
Center notification number inside navigation cart and wishlist.

File: Styles/nav.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    display:flex;  
+    align-items: center;           
+    background-color: rgb(209, 201, 201);
+
+}
+.left-nav{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width:700px;
```
Comment (Asuraking1n) [inline Styles/nav.css:16]: You could use % it helps in responsiveness

**Review by Asuraking1n — COMMENTED** [review]
Everything else looks good to me

File: Styles/nav.css (hunk 1 above)
Comment (ArpitPatni) [reply Styles/nav.css:16]: Thanks bro for suggestion.

**Pull request merged**